buildroot-external/patches/systemd/0003-systemd-timesyncd-delay-start-after-network-online.t.patch

```diff
@@ -0,0 +1,41 @@
+From 13cf1bb9c5fa91762184c3b0dddea1328c2746bc Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Jan=20=C4=8Cerm=C3=A1k?= <sairon@sairon.cz>
+Date: Wed, 11 Sep 2024 17:25:00 +0200
+Subject: [PATCH] systemd-timesyncd: delay start after network-online.target
+
+As explained in [1], it's desired for the systemd-timesyncd to run after
+the network is deemed online, otherwise the connectivity (at least on
+HAOS with NetworkManager) is not operational when the NTP sync is
+attempted and it fails, delaying the boot and leading to other problems.
+Because it's not possible to remove dependencies of existing units using
+drop-ins, patch the service template file for systemd-timesyncd unit
+instead, avoiding the need for complete unit file override as in [2].
```

Comment:
Shouldn't this include a delete of `systemd-timesyncd.service` then? :thinking:

Reply:
Yes, it absolutely should! 🤦🏻‍♂️I will amend the last commit to remove it. Thanks!